pkg/status/render/templates/process-agent.tmpl

```diff
@@ -5,69 +5,72 @@ NOTE: Changes made to this template should be reflected on the following templat
 =============
 Process Agent
 =============
-{{- if .error }}
+{{ with .processAgentStatus }}
```

Comment:
Same here for the header ?

Reply:
The code for rendering wasn't checking if the key `processAgentStatus ` was present like with `processAgentStatus`, so I did not place the `with` above the header. 

I guess we can still place it above the header, is just the status might change after. If customers are parsing the status output things might break